manim/mobject/geometry/line.py

```diff
@@ -1015,6 +1015,53 @@ def construct(self):
             return self.angle_value / DEGREES
         return self.angle_value
 
+    @staticmethod
+    def from_three_points(
+        A: np.ndarray, B: np.ndarray, C: np.ndarray, **kwargs
+    ) -> Angle:
+        """The angle between the lines AB and BC.
+
+        This constructs the angle :math:`\angle ABC`.
+
+        Parameters
+        ----------
+        A
+            The endpoint of the first angle leg
+        B
+            The vertex of the angle
+        C
+            The endpoint of the second angle leg
+
+        **kwargs
+            Further keyword arguments are passed to :class:`.Angle`
+
+        Returns
+        -------
+        The Angle calculated from the three points
+
+                    Angle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8),
+                    Angle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, other_angle=True),
+
+        Examples
+        --------
+        .. manim:: AngleFromThreePointsBasicExample
+            :save_last_frame:
+
+            class AngleFromThreePointsBasicExample(Scene):
+                def construct(self):
+                    right_angle = Angle.from_three_points(UP, ORIGIN, RIGHT)
```

Comment:
Looking at the rendered output, https://manimce--2684.org.readthedocs.build/en/2684/reference/manim.mobject.geometry.line.Angle.html?highlight=angle#anglefromthreepointsbasicexample, the angle only is a "right angle" in *some* sense. Either try changing the order of the legs (`RIGHT, ORIGIN, UP`), or additionally pass `other_angle=True`

Reply:
Sounds good! Changed the name!